python/samples/kernel-syntax-examples/azure_chat_gpt_with_data_api.py

```diff
@@ -1,61 +1,66 @@
 # Copyright (c) Microsoft. All rights reserved.
 
 import asyncio
-
+import logging
 import semantic_kernel as sk
 import semantic_kernel.connectors.ai.open_ai as sk_oai
 from semantic_kernel.connectors.ai.open_ai.contents.azure_chat_message_content import AzureChatMessageContent
 from semantic_kernel.connectors.ai.open_ai.contents.function_call import FunctionCall
 from semantic_kernel.connectors.ai.open_ai.contents.tool_calls import ToolCall
 from semantic_kernel.connectors.ai.open_ai.prompt_execution_settings.azure_chat_prompt_execution_settings import (
-    AzureAISearchDataSources,
+    AzureAISearchDataSource,
+    AzureAISearchDataSourceParameters,
     AzureChatPromptExecutionSettings,
-    AzureDataSources,
     ExtraBody,
 )
 from semantic_kernel.contents.chat_history import ChatHistory
 from semantic_kernel.contents.chat_role import ChatRole
 from semantic_kernel.functions.kernel_arguments import KernelArguments
 from semantic_kernel.prompt_template.input_variable import InputVariable
 from semantic_kernel.prompt_template.prompt_template_config import PromptTemplateConfig
-from semantic_kernel.utils.settings import (
-    azure_aisearch_settings_from_dot_env_as_dict,
-    azure_openai_settings_from_dot_env_as_dict,
-)
+from semantic_kernel.utils.settings import AzureChatCompletionSettings
 
 kernel = sk.Kernel()
+logging.basicConfig(level=logging.DEBUG)
 
 # Load Azure OpenAI Settings
-aoai_settings = azure_openai_settings_from_dot_env_as_dict()
+aoai_settings = AzureChatCompletionSettings()
 
 # For example, AI Search index may contain the following document:
 
 # Emily and David, two passionate scientists, met during a research expedition to Antarctica.
 # Bonded by their love for the natural world and shared curiosity, they uncovered a
 # groundbreaking phenomenon in glaciology that could potentially reshape our understanding of climate change.
 
-azure_ai_search_settings = azure_aisearch_settings_from_dot_env_as_dict()
+datasource_parameters_dict = AzureAISearchDataSourceParameters.from_dotenv().model_dump()
```

Comment:
Yeah, I think this would be a good idea to add, because I also had to figure things out from scratch while I was working on the samples.  :)  

Part of the setup can be achieved by following the DEV_SETUP.md instructions, but setup for the AOAI / OYD stuff specifically is a bit more involved.

Would it be okay to add this to a separate PR?  I may want to take some time to write up a more specific guide, but don't want to block the functional changes since the API retirement for 2023-12-01-preview is very soon.  LMK what you think -- I should be able to write something up for the samples sometime this week.

Reply:
yes, that works, thank you!